examples/individual_datastream/main.rs

```diff
@@ -0,0 +1,127 @@
+    )
+    .interface_directory("./examples/individual_datastream/interfaces")?
+    .ignore_ssl_errors();
+
+    // Create an Astarte Device (also performs the connection)
+    let mut device = astarte_device_sdk::AstarteDeviceSdk::new(&sdk_options).await?;
+    println!("Connection to Astarte established.");
+
+    // Create an thread to transmit
+    let device_cpy = device.clone();
+    tokio::task::spawn(async move {
+        loop {
+            // Send endpoint 1
+            let elapsed = now.elapsed().unwrap().as_secs() as i64;
+            device_cpy
+                .send(
+                    "org.astarte-platform.rust.examples.individual-datastream.DeviceDatastream",
+                    "/endpoint1",
+                    elapsed,
+                )
+                .await
+                .unwrap();
+            println!("Data sent on endpoint 1, content: {elapsed}");
+            // Sleep 1 sec
+            tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+            // Send endpoint 2
+            let elapsed = now.elapsed().unwrap().as_secs() as f64;
+            device_cpy
+                .send(
+                    "org.astarte-platform.rust.examples.individual-datastream.DeviceDatastream",
+                    "/endpoint2",
+                    elapsed,
+                )
+                .await
+                .unwrap();
+            println!("Data sent on endpoint 2, content: {elapsed}");
+            // Sleep 1 sec
+            tokio::time::sleep(std::time::Duration::from_secs(1)).await;
+        }
+    });
+
+    // Use the current thread to receive
+    loop {
+        match device.handle_events().await {
+            Ok(data) => {
+                if let astarte_device_sdk::Aggregation::Individual(var) = data.data {
+                    let mut parser_enable =
+                        delimited(tag::<_, _, (&str, ErrorKind)>("/"), digit1, tag("/enable"));
+                    let mut parser_intensity = delimited(
+                        tag::<_, _, (&str, ErrorKind)>("/"),
+                        digit1,
+                        tag("/intensity"),
+                    );
+                    if let Ok((_, led_id)) = parser_enable(&data.path) {
+                        println!(
+                            "Received new enable datastream for LED number {}. LED status is now {}",
+                            led_id.parse::<u16>().unwrap(),
+                            if var == true { "ON" } else { "OFF" }
+                        );
+                    }
+                    if let Ok((_, led_id)) = parser_intensity(&data.path) {
+                        let value: f64 = var.try_into().unwrap();
+                        println!(
+                            "Received new intensity datastream for LED number {}. LED intensity is now {}",
+                            led_id.parse::<u16>().unwrap(),
+                            value
+                        );
+                    }
```

Comment:
We could drop the nom dependency to simplify the example

```suggestion
                let mut iter = input.splitn(3, '/').skip(1);
                let led_id = iter.next().and_then(|id| id.parse::<u16>().ok())?;

                match iter.next() {
                    Some("enable") => {
                        println!(
                            "Received new enable datastream for LED number {}. LED status is now {}",
                            led_id,
                            if var == true { "ON" } else { "OFF" }
                        );
                    }
                    Some("intensity") => {
                        let value: f64 = var.try_into().unwrap();
                        println!(
                            "Received new intensity datastream for LED number {}. LED intensity is now {}",
                            led_id, value
                        );
                    }
                    _ => {}
                }
```